.changeset/gentle-toes-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+chore: added static method to duplicate predicate
```

Comment:
We need to update the changeset and PR title to reflect that this is a feature. Additionally, we are no longer adding a static method.